app/javascript/dashboard/components/CustomBrandPolicyWrapper.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { useStoreGetters } from 'dashboard/composables/store';
+import { computed } from 'vue';
+const props = defineProps({
+  showOnCustomBrandedInstance: {
+    type: Boolean,
+    default: true,
+  },
+});
+
+const getters = useStoreGetters();
+
+const isACustomBrandedInstance =
+  getters['globalConfig/isACustomBrandedInstance'];
+
+const shouldShowContent = computed(() => {
+  if (props.displayOnCustomBrandedInstance) {
+    return true;
+  }
+  return !isACustomBrandedInstance.value;
+});
```

Comment:
```suggestion
const { globalConfig: { isACustomBrandedInstance } } = useStoreGetters();

const shouldShowContent = computed(() => 
  props.showOnCustomBrandedInstance || !isACustomBrandedInstance.value
);
```

Reply:
This suggestion didn't work. The output useGetters is of the format `<module>/<getter>`. You can't use destructuring assignment here.